sig-service-catalog/charter.md

```diff
@@ -0,0 +1,133 @@
+    them to participate in related events, such as KubeCon.
+
+- Security Contacts
+  - Are a contact point for the Product Security Team to reach out to for triaging and handling of incoming issues.
+  - Must be a maintainer.
+  - Must accept the Kubernetes [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy).
+  - Defined in [SECURITY_CONTACTS](https://github.com/kubernetes-incubator/service-catalog/blob/master/SECURITY_CONTACTS) file.
+
+## Organizational management
+
+- SIG meets every week on Zoom at 1 PM PST on Mondays
+    - Agenda [here](https://docs.google.com/document/d/17xlpkoEbPR5M6P5VDzNx17q6-IPFxKyebEekCGYiIKM/edit#).
+    - Anyone is free to add new agenda items to the doc.
+    - Recordings of the calls are made available [here](https://goo.gl/ZmLNX9).
+- SIG members explicitly representing the group at conferences (SIG progress reports, deep dives, etc.)
+  should make their slides available for perusal and feedback at least 2 week in advance.
+- [Working groups](https://docs.google.com/document/d/1fghxARW-doHrNmBYCijhODsGoIFawpeU4X0end-XnUI/edit#) can be initiated by any member. To create a new one, add the topic
+  to the weekly call’s agenda for discussion.
+  - These are not the same as cross-SIG working groups.
+  - Working groups exist for an unspecified period of time, so that interested members
+    can meet to discuss and solve problems for our SIG.
+
+### Project management
+- Milestones are defined by SIG maintainers.
+- Anyone is free to request for a discussion of the milestones/plans during
+  a weekly call.
+- Weekly releases are typically done on Thursdays, and any member who has
+  maintainer rights is free to initiate it. _Friday releases are strongly discouraged._
+- Major releases are planned and discussed among the SIG members during
+  regular weekly calls.
+- The release process is defined [here](https://github.com/kubernetes-incubator/service-catalog/wiki/Release-Process).
+- Anyone can request to work on an issue by commenting on it with `#dibs`.
+
+
+### Technical processes
+- All technical decisions are made either through issues, pull requests or
+  discussions during the weekly SIG meeting. Major decisions should be documented
+  in an issue or pull request.
+- There is no requirement that all pull requests have an associated issue.
+  However, if the PR represents a significant design decision then it is
+  recommended that it be discussed among the group to avoid unnecessary
+  coding that might not be accepted.
+- While everyone is encouraged to contribute to the discussion of
+  any topic, ultimately any changes made to the codebase must be
+  approved by the maintainers.
+- Pull requests are required to have at least 2 maintainers approve them using the `LGTM 1` and `LGTM 2` labels.
+- Pull requests that are labeled with `do not merge` or have an on-going
```

Comment:
we should add the `/hold` prow plugin and consider using it for this marking.

Reply:
👍  My plan is to do a second pass at the charter after we move to PROW. I'll add an issue for that to the PROW milestone.

I don't want to have getting a charter in place mixed up with stuff that hasn't happened yet.

Reply:
good news! https://github.com/kubernetes/test-infra/commit/ad64db9bc945dce9bd618acd8a197d714c41de05

Reply:
HOT, let me play with it on an issue to make sure I got it right and I'll put it in.